cms/templates/js/staff-lock-editor.underscore

```diff
@@ -6,7 +6,49 @@
         <%- gettext('Section Visibility') %>
     <% } %>
 </h3>
-<div class="modal-section-content staff-lock">
+<% if (enableHideFromTOCUI) { %>
+<div class="list-fields list-input content-visibility" role="group" aria-labelledby="content_visibility_label">
+  <div class="modal-section-content no-visibility-option">
+      <label class="label">
+          <input type="radio" id="visible_to_learners" name="content-visibility" class="input input-radio" />
+          <span> <%- gettext('Visible to learners') %> </span>
+      </label>
+      <p class='field-message' id='staff-lock-description'> <%- gettext('It is visible to learners, ensuring that all learners can view its contents.') %> </p>
+  </div>
+    <div class="modal-section-content staff-lock">
```

Comment:
It looks like you are duplicating this whole node. It is a choice to either duplicate part of the template or have more `if (enableHideFromTOCUI)` checks.

Would you consider having the following approach?
```
if (enableHideFromTOCUI)
   ... the first block this feature adds
endif

   <!-- the unmodified section-content -->
   <div class="modal-section-content staff-lock">

if (enableHideFromTOCUI)
   ... the second block this feature adds
endif 


```

Reply:
Yes, I'll consider it. Thank you.

Reply:
This part uses radio inputs instead of checkboxes, so we need these modifications. We could specify which input type to use instead. I'll try that :). I'll let you know when it's done

Reply:
Done! Could you check again? Thanks

Reply:
The refactor broke some styles, I'll be working on fixing the issues. Thanks for your patience.

Reply:
Fixed.